website/docs/api/type-aliases/Permission.md

```diff
@@ -6,7 +6,8 @@ type Permission =
   | "data:read"
   | "data:read_write"
   | "data:delete"
-  | "project:delete";
+  | "project:delete"
+  | "backups:read";
```

Comment:
Oops, did I miss this? Well spotted.

Reply:
The thing is that the docusaurus docs in this repo `website` folder have to be rebuilt for the doc files to be updated. This is not necessary though for changes to be reflected in https://doist.github.io/todoist-api-typescript/, as the `deploy-docusaurus` github action takes care of that.